package.json

```diff
@@ -13,23 +13,23 @@
   },
   "scripts": {
     "server": "http-server . -p 8002 -o playground/index.html",
-    "postinstall": "gulp build-debug-infos && npm run build-declaration && npm run bundle-adapter && npm run fetch-ral && npm run install:typedoc-plugin && npm run install:native-pack-tool",
+    "postinstall": "gulp build-debug-infos && npm run build-declaration && npm run bundle-adapter && npm run fetch-ral && npm run install:native-pack-tool",
     "build": "gulp build",
     "build:dev": "gulp build-h5-source",
     "build:min": "gulp build-h5-minified",
     "build-declaration": "npm run build-const && gulp build-declarations",
     "build-api-json": "gulp build-api-json -i index.ts -o ./3d-api-docs/tempJson",
     "build-api": "npm run build-api-json && gulp build-3d-api -i index.ts -o ./3d-api-docs -j ./3d-api-docs/tempJson",
     "checkout-engine-version": "node ./scripts/checkout-engine-version.js",
-    "varify-engine-version": "node ./scripts/varify-engine-version.js",
+    "verify-engine-version": "node ./scripts/verify-engine-version.js",
     "fetch-ral": "node ./scripts/fetch-ral.js",
     "bundle-adapter": "node ./scripts/bundle-adapter.js",
-    "build-const": "node ./scripts/build-consts.js",
+    "build-const": "node ./scripts/build-const.js",
     "clear": "node ./scripts/clear-cache.js",
     "clear-platform": "node ./native/utils/clear-platform.js && node ./scripts/clear-file.js",
     "update:native-external": "node ./native/utils/update-external.js",
     "test": "gulp test",
-    "install:typedoc-plugin": "cd ./scripts/typedoc-plugin && npm install --only=production",
+    "generate-doc-info": "node ./scripts/generate-doc-info",
```

Comment:
need update the api repo`s README.md

Reply:
There is no explanation for this part in readme

Reply:
http://git.cocos.org:9080/G_Engine/cocos3d-docs-api/-/tree/v3.8.1 in this repo, we have a readme  show how to release a api doc.